.github/workflows/main.yml

```diff
@@ -5,10 +5,8 @@ name: CI
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
-  push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, 'week**', 'test**', 'task**']
```

Comment:
maybe just write `on: pull_request` since it seems like we're calling it for basically every branch. we could get some unexpected behavior and fail to run if someone names a branch incorrectly

Reply:
Yeah that makes more sense. I'll make a new pull request and update the corresponding docs.